ui/pages/reset_email.tsx

```diff
@@ -120,6 +121,13 @@ const ResetEmail: NextPage = () => {
     } as RecoveryFlow;
   };
 
+  flow?.ui.nodes.map((node) => {
+    if (isContinueWithPasswordReset(node)) {
+      node.meta.label.text = "Submit";
```

Comment:
So we set the text value to `Submit` so that this later gets handled by https://github.com/canonical/identity-platform-login-ui/blob/main/ui/pages/reset_email.tsx#L100?

Why don't we just change the check there? (IMHO `Continue` is better than `Reset password`, as the user already pressed the reset password button).

Reply:
I am not sure what https://github.com/canonical/identity-platform-login-ui/blob/main/ui/pages/reset_email.tsx#L100 does exactly, but it doesn't change the value of the [step 2](https://github.com/canonical/identity-platform-login-ui/blob/main/ui/tests/__screenshots__/reset-password.spec.ts/reset-password-from-oidc-app-2.png) of the recovery flow, which I covered with this small refactor

Reply:
If I understand correctly it uses the value of the `node.meta.label.text` to make a decision about whether a Back button is needed. AFAICT this is why the tests were originally broken, because the text value was changed from `Continue` to `Submit`.

All I am saying is that I am not sure if we want to add this mapping, just to keep the old texts on the buttons.

Reply:
The continue button jumping places is not ideal, I'll give a suggestion in tomorrow first thing!

Reply:
This is my proposal:
Back in both pages, resend code as a secondary CTA, as it does not move across pages, and a consistent main CTA to "Reset Password" as that's the main goal of a user going through this flow.
![image](https://github.com/user-attachments/assets/555fc419-1dd9-4e63-b2a4-1b9f54d29e47)
![image](https://github.com/user-attachments/assets/b10c5ac9-1055-4628-bb91-11d259679fd1)

Reply:
Thanks for the input @lukasSerelis.
I created an issue https://github.com/canonical/identity-platform-login-ui/issues/376 to improve the UI, meanwhile I'll keep the current design to keep the PR focused on the sdk upgrade.